Application/Frontend/__tests__/FriendsTab.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { MongoClient } from 'mongodb';
+import handler from '@/pages/api/FriendsTab'; 
+import { NextApiRequest, NextApiResponse } from 'next';
+
+// Mock MongoClient and dbCollection
+jest.mock('mongodb', () => ({
+  MongoClient: jest.fn()
```

Comment:
It might make more sense to actually extract the data directly from MongoDB instead of constructing a mock database / collection. Although the focus for this test is on checking if you can extract the friend list of a particular user, actually running that query with the database makes more sense as that is how the application would actually handle it.